### PR TITLE
Dev-14-create-main-page-of-website

### PR DESCRIPTION
**Pull request by MunzerDw**
todo:
- find better header animation 
- add correct logos in logos sections

### CODE REVIEW

File: components/Logos.tsx [hunk 1]
```diff
@@ -3,7 +3,7 @@ import Link from "next/link";
 import Section from "./ui/Section";
 
 export interface Props {
-  title?: string;
+  title?: string | React.ReactNode;
```
Comment (simon-hng) [inline components/Logos.tsx]: I would prefer to just use the children prop or only accept `ReactNode`

File: components/VerticalCards.tsx [hunk 2]
```diff
@@ -0,0 +1,37 @@
+import Image from "next/image";
+import React from "react";
+
+export interface Props {
```
Comment (simon-hng) [inline components/VerticalCards.tsx:6]: The props type should refer to the main export of this file. In this case
```ts
  card: {
    img: string;
    title: string;
    text: string;
    subtext?: string;
    body: React.ReactNode;
  }[];
```

File: components/VerticalCards.tsx [hunk 3]
```diff
@@ -0,0 +1,37 @@
...
+  data: {
+    img: string;
+    title: string;
+    text: string;
+    subtext?: string;
+    body: React.ReactNode;
+  };
+}
+
+export default function VerticalCards({ data }) {
```
Comment (simon-hng) [inline components/VerticalCards.tsx]:
```suggestion
export default const VerticalCards: FC<Props> = ({ data }) => {
```

File: components/ui/Button.tsx [hunk 4]
```diff
@@ -16,18 +17,32 @@ export interface Props
...
+  link?: boolean;
+  href?: string;
   children: React.ReactNode;
 }
 
 export default function Button({
   intent,
   children,
   className,
+  link,
```
Comment (simon-hng) [inline components/ui/Button.tsx]: use @radix-ui/react-slot and composition

https://github.com/tum-ai/react-components/blob/main/src/components/Button.tsx

File: pages/index.tsx [hunk 5]
```diff
@@ -0,0 +1,413 @@
...
+import * as THREE from "three";
+
+export default function Index() {
+  return (
+    <>
+      <Hero />
+      <Content1 />
+      <Content2 />
+      <div className="relative h-[450px] w-full bg-blend-overlay">
+        <div className="bg-primary-500 absolute z-10 h-full w-full opacity-20"></div>
```
Comment (simon-hng) [inline pages/index.tsx]:
```suggestion
        <div className="bg-primary-500 absolute z-10 h-full w-full opacity-20" />
```

File: pages/index.tsx [hunk 6]
```diff
@@ -0,0 +1,413 @@
...
+        ]}
+      />
+    </Section>
+  );
+}
+
+function Content2() {
+  return (
+    <Section>
+      {/* title */}
```
Comment (simon-hng) [inline pages/index.tsx]:
```suggestion

```

File: pages/index.tsx [hunk 7]
```diff
@@ -0,0 +1,413 @@
...
+      <h2 className="text-uppercase w-full text-center text-4xl font-bold">
+        With over 170 active members, TUM.ai is{" "}
+        <span className="text-purple-500">{`Germany's leading AI student initiative`}</span>{" "}
+        , located in Munich.
+      </h2>
+
+      <hr className="my-16 border-gray-500" />
+
+      <div className="grid gap-16 lg:grid-cols-2">
+        {/* image */}
```
Comment (simon-hng) [inline pages/index.tsx:218]:
```suggestion

```

File: pages/index.tsx [hunk 8]
```diff
@@ -0,0 +1,413 @@
...
+        <div className="relative h-[450px] w-full rounded bg-blend-overlay">
+          <div className="absolute z-10 h-full w-full bg-purple-500 opacity-20"></div>
+          <Image
+            src={"/assets/aibootcamp.jpg"}
+            alt=""
+            fill
+            className="object-cover"
+          />
+        </div>
+        {/* text */}
```
Comment (simon-hng) [inline pages/index.tsx:228]:
```suggestion

```

File: pages/index.tsx [hunk 9]
```diff
@@ -0,0 +1,413 @@
...
+              " we aim at lowering the entry-bar to AI-creation and -usage for
+              people of every domain by establishing a platform to make
+              practical experience through a diverse set of applied AI projects.
+              We therefore educate and connect students from diverse backgrounds
+              to incentivize new interdisciplinary AI projects and push the
+              creation and usage of applicable and safe AI in all domains.{" "}
+            </p>
+          </div>
+        </div>
+        {/* card */}
```
Comment (simon-hng) [inline pages/index.tsx:270]:
```suggestion

```

File: pages/index.tsx [hunk 10]
```diff
@@ -0,0 +1,423 @@
...
+      <div className="flex h-full w-full items-center text-white">
+        <div className="m-auto space-y-8 lg:inline-block">
+          <Image
+            src={"/assets/logo_new_white_standard.png"}
+            alt="Logo"
+            width={450}
+            height={100}
+          />
+          <div className="m-auto flex w-fit flex-col items-center justify-center space-y-4 lg:flex-row lg:space-x-4 lg:space-y-0">
+            <Button
```
Comment (simon-hng) [inline pages/index.tsx]: Those are not clickable

File: components/Logos.tsx (hunk 1 above)
Comment (simon-hng) [reply components/Logos.tsx]: Actually does not belong in this component at all

**Review by MunzerDw — COMMENTED** [review]
great, now we can deploy it!

**Comment by MunzerDw** [comment]
I can approve since its my PR but we can merge

**Pull request merged**